### DISCUSSION TITLE
email blade from db table

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#450] [Q&A (Help)]
Is there any add-on to manage email templates from database, perhaps with the possibility of using variables in the text of the template? :)

**Comment by tabacitu** [comment]
No, no such add-on right now. Let us know if you create one!

**Comment by munjaldevelopment** [comment]
Hi @blondie63 You can explore this git repo: `https://github.com/Snowfire/Beautymail`. I have used this repo in few projects & its pretty good. It give option to customize "**email template**" also.

@tabacitu let me know if you want me to help on this?

**Reply by tabacitu** [reply]
Oh I like Beautymail - I've used it in a few projects too. I think it's 50% of what @blondie63 requested, though. To me it sounds like he wants _an interface_ to manage those templates. That would be cool, but we can't make it a priority - it's the first time someone asks for this.
